compiler/noirc_evaluator/src/ssa/acir_gen/mod.rs

```diff
@@ -84,6 +94,34 @@ impl SharedContext {
     fn new_generated_pointer(&self) -> u32 {
         self.generated_brillig.len() as u32
     }
+
+    fn generated_brillig_stdlib_pointer(
+        &self,
+        brillig_stdlib_func: &BrilligStdlibFunc,
+    ) -> Option<&u32> {
+        self.brillig_stdlib_func_pointer.get(brillig_stdlib_func)
```

Comment:
```suggestion
    ) -> Option<u32> {
        self.brillig_stdlib_func_pointer.get(brillig_stdlib_func).copied()
```
Might as well copy here